app/assets/javascripts/admin/routes/admin-site-settings.js.es6

```diff
@@ -10,9 +10,10 @@ export default Discourse.Route.extend({
   },
 
   afterModel(siteSettings) {
-    this.controllerFor("adminSiteSettings").set(
-      "allSiteSettings",
-      siteSettings
-    );
+    const controller = this.controllerFor("adminSiteSettings");
+
+    if (!controller.get("visible")) {
+      controller.set("visible", siteSettings);
+    }
```

Comment:
I really want to move this to the constructor in `controllers/admin-site-settings.js.es6`, but then  `visible` wouldn't be initialized in time for `routes/admin-site-settings-index.js.es6`:

https://github.com/discourse/discourse/blob/e10236cb6863777c4eb96765b5fb70e1965afa3f/app/assets/javascripts/admin/routes/admin-site-settings-index.js.es6#L9